### DISCUSSION TITLE
Compatibility with deprecated `cancellable` argument to anyio

### DISCUSSION THREAD

**Discussion by DaniilAnichin** [fastapi/asyncer#144] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the Asyncer documentation, with the integrated search.
- [X] I already searched in Google "How to X in Asyncer" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to Asyncer but to [AnyIO](https://github.com/agronholm/anyio).
- [X] I already checked if it is not related to Asyncer but to [Trio](https://github.com/python-trio/trio).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
import time
import warnings

warnings.simplefilter('always')

import anyio
from asyncer import asyncify


def do_sync_work(name: str):
    time.sleep(1)
    return f'Hello, {name}'


async def main():
    message = await asyncify(do_sync_work)(name='World')
    print(message)


anyio.run(main)
```

### Description

* Whenever anyio [>=0.4.1](https://github.com/agronholm/anyio/releases/tag/4.1.0) is installed alongside asyncer, warnings about deprecated parameter are generated
* While warnings itself are not an issue, it will become as anyio will most probably remove outdated parameter at some moment
* Will it make sense to make a new release of asyncer, with pin of asyncio >= 0.4.1, utilizing new parameter name (cancellable -> abandon_on_cancel)?

### Operating System

Linux

### asyncer Version

0.0.5

### Python Version

3.11.2

### Additional Context

anyio affected versions are >=0.4.1, released Nov 23

**Comment by tiangolo** [comment, marked as answer]
Thanks! This was solved in https://github.com/fastapi/asyncer/pull/202 :nerd_face: 

This is available in Asyncer 0.0.8, just released. :tada:
